backend/models/beneficiary_model.py

```diff
@@ -18,52 +17,59 @@
 class Beneficiary(Document):
     first_name = StringField(max_length=500)
     last_name = StringField(max_length=500)
-    phone = IntField(min_value=16, max_value=120)
-    landline = IntField(min_value=16, max_value=120)
+    phone = StringField()
+    landline = StringField()
     age = IntField(min_value=16, max_value=120)
-    zone = StringField(choices=[zone.value for zone in Zone], required=True)
+    zone = StringField(choices=[zone.name for zone in Zone], required=True)
```

Comment:
why you did change it?

Reply:
We want to use field name of enum, not label. https://github.com/code4moldova/voluntar-web/pull/137/files#diff-7c4276a86b2d0c9995ca3d396372bd9fR4